<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testTableWidget</class>
 <widget class="QWidget" name="testTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="SPRSpectrumGraphicsWidget" name="graphic" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="gist">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="kspectTable">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>250</y>
     <width>751</width>
     <height>121</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>№ куска</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ручей</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>H1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>H2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>H3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>t. изм. мс</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="cbThread">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>thread 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>thread 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>thread 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>thread 4</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="stoneTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>1141</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>i_prg</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>p_prg</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>p_tk</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>p_tkH1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>p_tkH2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>p_tkH3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>w_count</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>s_rst1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>s_rst2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>s_rst3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>s_rst4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>s_rst5</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>error</string>
    </property>
   </column>
  </widget>
  <widget class="SPRSpectrumListTable" name="table">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>201</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bGetGist">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>GET GIST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bGETSepar">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>GET SEPAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bGetKSPK">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get KSPK</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>SPRSpectrumListTable</class>
   <extends>QTableWidget</extends>
   <header>sprspectrumlisttable.h</header>
  </customwidget>
  <customwidget>
   <class>SPRSpectrumGraphicsWidget</class>
   <extends>QWidget</extends>
   <header>sprspectrumgraphicswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
